src/main/java/kitchenpos/menu/application/MenuService.java

```diff
@@ -0,0 +1,72 @@
+package kitchenpos.menu.application;
+
+import kitchenpos.menu.domain.*;
+import kitchenpos.menu.dto.MenuRequest;
+import kitchenpos.menu.dto.MenuResponse;
+import kitchenpos.menu.exception.NotFoundMenuGroupException;
+import kitchenpos.menu.exception.NotFoundProductException;
+import kitchenpos.wrap.Price;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@Transactional
+public class MenuService {
+    private final MenuRepository menuRepository;
+    private final MenuGroupRepository menuGroupRepository;
+    private final ProductRepository productRepository;
+
+    public MenuService(MenuRepository menuRepository, MenuGroupRepository menuGroupRepository, ProductRepository productRepository) {
+        this.menuRepository = menuRepository;
+        this.menuGroupRepository = menuGroupRepository;
+        this.productRepository = productRepository;
+    }
+
+    public MenuResponse create(final MenuRequest menuRequest) {
+        MenuGroup menuGroup = menuGroupRepository.findById(menuRequest.getMenuGroupId()).orElseThrow(() -> new NotFoundMenuGroupException());
```

Comment:
```suggestion
        MenuGroup menuGroup = menuGroupRepository.findById(menuRequest.getMenuGroupId())
                .orElseThrow(NotFoundMenuGroupException::new);
```

한줄보다는 예외처리에 엔터를 넣어주면 어떤 처리를 하는지 파악하기 더 좋을 것 같네요

Reply:
파악하기 쉽게 라인 변경적용했습니다~😀😀